spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -236,6 +268,68 @@
           end
         end
       end
+
+      context "with available user roles restricted by type of admin" do
+        let!(:opendoors_role) { create(:role, name: "opendoors") }
+        let!(:archivist_role) { create(:role, name: "archivist") }
+        let!(:tag_wrangler_role) { create(:role, name: "tag_wrangler") }
+
+        before do
+          admin.update!(roles: ["open_doors"])
+        end
```

Comment:
Could this be a single line ` before { admin.update!(roles: ["open_doors"]) }` or does Reviewdog complain about that?